mwdb/core/auth.py

```diff
@@ -0,0 +1,48 @@
+import datetime
+
+import jwt
+
+from mwdb.core.config import app_config
+
+
+def generate_token(fields, scope, expiration=24 * 3600):
+    time_claims = {
+        "exp": datetime.datetime.now(tz=datetime.timezone.utc)
+        + datetime.timedelta(seconds=expiration),
```

Comment:
Formatting is more readable if we put long expressions in parentheses (I'm not 100% sure, so check it with black)
```suggestion
        "exp": (
            datetime.datetime.now(tz=datetime.timezone.utc)
            + datetime.timedelta(seconds=expiration)
        ),
```